task/sast-coverity-check/0.2/patch.yaml

```diff
@@ -460,7 +460,7 @@
         set -x
         UPLOAD_FILE="$1"
         MEDIA_TYPE="$2"
-        oras attach --no-tty --registry-config "${HOME}/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}" "${UPLOAD_FILE}:${MEDIA_TYPE}"
+        oras attach --no-tty --registry-config "${HOME}/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}@${IMAGE_DIGEST}" "${UPLOAD_FILE}:${MEDIA_TYPE}"
```

Comment:
This will not work because the parameter was already removed in 8b62b416ce3332cf998d24c32d00061188d22f9c.  It needs to be reintroduced in the task, in the default build pipeline template, and a new version of the task (and migration to it) will be needed.

Reply:
Yes, I think we need to create `task/sast-coverity-check/0.3` to ensure smooth migration.